Composer/packages/client/src/pages/knowledge-base/QnAPage.tsx

```diff
@@ -34,10 +51,26 @@ const QnAPage: React.FC<RouteComponentProps<{
   const actions = useRecoilValue(dispatcherState);
   const dialogs = useRecoilValue(dialogIdsState(actualProjectId));
   const qnaFiles = useRecoilValue(qnaFilesState(actualProjectId));
-  //To do: support other languages
-  const locale = 'en-us';
-  //const locale = useRecoilValue(localeState);
-  const creatQnAOnInfo = useRecoilValue(createQnAOnState);
+  const locale = useRecoilValue(localeState(actualProjectId));
+  const createQnAOnInfo = useRecoilValue(createQnAOnState);
+  const settings = useRecoilValue(settingsState(actualProjectId));
+  const { defaultLanguage } = settings;
+  const languages = useMemo(() => (defaultLanguage === locale ? [defaultLanguage] : [locale, defaultLanguage]), [
+    locale,
+    defaultLanguage,
+  ]);
```

Comment:
It's not necessarily to wrap this statement in a useMemo hook since the calculation is not very expensive.

Reply:
remove useMemo